cmd/get_peers_list.go

```diff
@@ -0,0 +1,35 @@
+package main
```

Comment:
Can you maybe put this file inside `test` directory and then put there `test.sh` as well, so it'll be clear that it's used only for tests?